circle.yml

```diff
@@ -28,8 +28,7 @@ test:
     - cd "$PROJECT_PATH" && set -o pipefail && make test_integrations 2>&1 | tee test_integrations.log:
         timeout: 1800
   post:
-    - cd "$PROJECT_PATH" && mv test_integrations.log "${CIRCLE_ARTIFACTS}"
-    - cd "$PROJECT_PATH" && bash <(curl -s https://codecov.io/bash) -f coverage.txt
-    - cd "$PROJECT_PATH" && mv coverage.txt "${CIRCLE_ARTIFACTS}"
-    - cd "$PROJECT_PATH" && cp test/logs/messages "${CIRCLE_ARTIFACTS}/docker.log"
```

Comment:
I think we still need this log

Reply:
The docker log usually isn't very telling, what information would we lose?

Reply:
> The docker log usually isn't very telling, what information would we lose?

huh? all docker containers are logging msgs to `docker.log` through syslog

Reply:
Yeh the eg. p2p integration tests spin up 4 containers and do a bunch of tests with them. The only way to see what's happening on those nodes is the docker.log (we don't pipe it out to circle stdout directly because its so verbose)

Reply:
Surely, but the majority of those lines are the output of our binaries which we captured with the `test_integration.log`.

Reply:
Cool, clarified now. Would be nice to have a better split of those logs for easier tracing when doing forensics.